src/hooks/mediaHooks/useGetMultipleMediaHook.tsx

```diff
@@ -0,0 +1,50 @@
+import { useQuery, UseQueryOptions, UseQueryResult } from "react-query"
+
+import { MEDIA_MULTIPLE_CONTENT_KEY } from "constants/queryKeys"
+
+import { apiService } from "services/ApiService"
+
+import { MediaService } from "services"
+import { MediaData } from "types/directory"
+import { MultipleMediaParams } from "types/media"
+
+const getMultipleMedia = (params: MultipleMediaParams) => {
+  const mediaService = new MediaService({ apiClient: apiService })
+
+  return Promise.all(
+    params.mediaSrcs.map((mediaSrc) => {
+      const mediaUrlTokens = mediaSrc.split("/")
+      // Note: The mediaUrl will always start with a "/"
```

Comment:
Hmm it is more of a note although we are somewhat guaranteed about this behaviour. I'm not really inclined to assert here since it is safely ignored when the preview is rendered.

Reply:
hmm if we are not sure, should we still put this as comments ah?